libs/SalesforceReact/package.json

```diff
@@ -18,7 +18,7 @@
         "@react-native-community/eslint-config": "^2.0.0",
         "@types/jest": "^26.0.24",
         "@types/react": "^18.0.21",
-        "@types/react-native": "^0.70.6",
+        "@types/react-native": "^0.70.13",
```

Comment:
I don't see a 0.70.13 for @types/react-native. See [here](https://www.npmjs.com/package/@types/react-native?activeTab=versions).